proposals/2021-05-gravity-bridge-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+# Signaling Proposal - Deployment of Gravity Bridge on the Cosmos Hub
+
+![Gravity Bridge](https://raw.githubusercontent.com/althea-net/cosmos-gravity-bridge/main/gravity-bridge.svg)
+
+## Summary
+
+This proposal is a Request For Comment to the ATOM community regarding the activation of the Gravity Bridge module onto the Cosmos Hub.
+
+By voting YES to this proposal, you will signal that you approve of having the Gravity Ethereum <> Cosmos bridge deployed onto the Cosmos Hub.
+
+## Vision
+
+Gravity as an Ethereum-Cosmos bridge is designed for the Cosmos Hub to pull as much value as possible into the orbits of Cosmos via a direct and decentralized bridge. Gravity will be able to bring ERC20 assets from Ethereum into Cosmos, as well as Cosmos assets to Ethereum ERC20 representations. ATOM, and any other asset in the Cosmos ecosystem, will be able to trade on Uniswap and other Ethereum AMMs, and interact with Ethereum DeFi like any ERC20 token. As well, Cosmos Hub users can use the Gravity DEX to trade between ERC20 assets and tokens that are transferred using IBC. This will bring a tremendous amount of liquidity and utility to these multi-chain assets.
+
+## Cosmos, Ethereum, and Gravity
+
+Gravity is a secure and highly efficient bridge between EVM and Cosmos SDK-based blockchains. At a high-level, Gravity enables token transfers from Ethereum to the Cosmos Hub and back again by locking tokens on the Ethereum side and minting equivalent tokens on the Cosmos side.
+
+Gravity is completely non-custodial. Control of the bridge mirrors the active validator set on the Cosmos SDK-based chain, and validator stake on Cosmos can be slashed for misbehavior involving the Gravity bridge.
+
+The Gravity Ethereum contract is highly optimized, utilizing batches to dramatically reduce the cost of Cosmos -> Ethereum transfers. Sending funds from Cosmos back to Ethereum can be up to 50% less costly than a normal ERC20 send.
+
+## How do validators support the Gravity Bridge?
+
+Cosmos Hub validators will run three key software components of the Gravity bridge:
+
+- The Gravity bridge module, integrated into gaiad (the binary that runs the Cosmos Hub)
+- The Gravity bridge Orchestrator
+- A Geth light client or any Ethereum full node implementing the JSON-rpc standard
+
+### Cosmos to Ethereum:
+
+To send transactions from Cosmos to Ethereum, the Gravity Bridge module of a validator's Gaia instance first packages the transaction data, and makes it available on an endpoint. The Orchestrator then signs this data with the validator’s Ethereum key, and submits it as a message to the Ethereum network. The Ethereum signature of each Cosmos validator needs to be assembled and submitted to the Ethereum chain by relayers.
+
+Validators may be slashed if they fail to submit Ethereum signatures within 6000 Cosmos Hub blocks (about eight hours) of their creation.
```

Comment:
Would be good to mention this in comparison to the existing liveness slashing rule (ie. failing to sign for 9500 blocks, or more accurately failing to sign at least 500 of the last 10,000 blocks):

```
$ gaiad query slashing params
min_signed_per_window: "0.050000000000000000"
signed_blocks_window: "10000"
```

Reply:
added a comparison